api/http/management.go

```diff
@@ -27,3 +37,66 @@ func NewManagementAPI(app app.App) *ManagementAPI {
 		App: app,
 	}
 }
+
+func (api *ManagementAPI) ConfigurationSet(c *gin.Context) {
+	var configuration model.Configuration
+
+	ctx := c.Request.Context()
+	devID, err := uuid.Parse(c.Param("device_id"))
+	if err != nil {
+		rest.RenderError(c,
+			http.StatusBadRequest,
+			errors.Wrap(err, "correctly formatted device id is needed"),
+		)
+		return
+	}
+
+	err = c.ShouldBindJSON(&configuration)
+	if err != nil {
+		rest.RenderError(c,
+			http.StatusBadRequest,
+			errors.Wrap(err, "malformed request body"),
+		)
+		return
+	}
+
+	err = api.App.ConfigurationSet(ctx, devID, configuration)
+	if err != nil {
+		switch cause := errors.Cause(err); cause {
+		case store.ErrDeviceAlreadyExists:
+			rest.RenderError(c, http.StatusConflict, cause)
+		default:
+			c.Error(err) //nolint:errcheck
+			rest.RenderError(c,
+				http.StatusInternalServerError,
+				errors.New(http.StatusText(http.StatusInternalServerError)),
+			)
+		}
+		return
+	}
+	c.Status(http.StatusCreated)
+}
+
+func (api *ManagementAPI) ConfigurationGet(c *gin.Context) {
```

Comment:
Not super important, but I would suggest VerbNoun naming, thus `GetConfiguration`

Reply:
changed.